dss_datamover/tests/requirements.txt

```diff
@@ -0,0 +1,4 @@
+pytest
```

Comment:
Better to have version numbers like pytest == x.y.z
Sometimes the newer versions or older versions will run into compat issues.
But the downside of this approach is, we need to update this file when we want to move to newer versions after some regression testing.